src/deployer.ts

```diff
@@ -76,7 +76,7 @@ export abstract class Deployer extends EventEmitter {
   /**
    * Deploy the app.
    */
-  public abstract deploy(): Promise<void>;
+  public abstract deploy(): Promise<void | number>;
```

Comment:
Is there any chance there's more information we'd want returned in the future? Like an error message? Or a list of deployed things?

If so, I think we should return `{ exitCode: number }` so that we can expand it if necessary